### PR TITLE
Feature/jsonld

### PR DESCRIPTION
**Pull request by steffansluis**
This PR changes the graph broker to accept and ouput JSON-LD documents. It also adds tests. For reasons to do with CircleCI 2.0 and `docker-compose`, the `feedbackfruits-knowledge-bus` is not used in the CI. A presumably identical Kafka bus serves to take its place for now.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require('dotenv').load({ silent: true });
+
+const {
+  NAME = 'feedbackfruits-knowledge-graph-broker',
+  CAYLEY_ADDRESS = 'http://localhost:64210',
+  KAFKA_ADDRESS = 'tcp://localhost:9092',
+  INPUT_TOPIC = 'update_requests',
+  OUTPUT_TOPIC = 'updates',
+  CONCURRENCY = '100',
```
Comment (joostverdoorn) [inline src/config.ts]: Let's make sure this is a number

File: src/doc.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export type Doc = {
+  '@id': string
+};
```
Comment (joostverdoorn) [inline src/doc.ts]: As memux doesn't require things to be jsonld docs, we maybe should put this type else (context?). Could potentially be done later.

File: src/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        [decodeIRI(predicate)]: [
+          ...((memo[decodeIRI(subject)] && memo[decodeIRI(subject)][decodeIRI(predicate)]) || []),
+          object
+        ]
+      }
+    };
+  }, {}));
+};
+
+export const docToQuads = async (doc: Doc): Promise<Quad[]> => {
```
Comment (joostverdoorn) [inline src/helpers.ts]: `Array<Quad>`

File: src/helpers.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  return new Promise<Quad[]>((resolve, reject) => {
+    jsonld.toRDF(doc, { format: 'application/nquads' }, function(err, nquads) {
+      if (err != null) return reject(err);
+      // console.log('Parsing nquads:', nquads);
+      const lines = nquads.split('\n');
+      lines.pop() // Remove empty newline
+      const quads = lines.map(line => {
+        // A line looks like: `<<http://some.domain/janedoe>> <<http://schema.org/jobTitle>> "\\"Professor\\"" .␊
```
Comment (joostverdoorn) [inline src/helpers.ts]: double check

File: src/config.ts (hunk 1 above)
Comment (steffansluis) [inline src/config.ts]: :+1:

File: src/doc.ts (hunk 2 above)
Comment (steffansluis) [inline src/doc.ts]: I agree. Will take this into account once more things get abstracted away to context.

File: src/helpers.ts (hunk 3 above)
Comment (steffansluis) [inline src/helpers.ts]: Do you prefer `Array<T>` over `T[]`? TSLint has [options](https://palantir.github.io/tslint/rules/array-type/) for this.

**Pull request merged**